packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.scss

```diff
@@ -254,4 +254,14 @@ $DropDown-item-height: 36px;
 .divider {
   height: 1px;
   background-color: $dividerColor;
+}
+
+.span{
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  min-width: 100px;
```

Comment:
you can set `min-width: 0px;` instead. Flexbox just needs a min-width set so that it can shrink things!